corehq/apps/app_manager/views/forms.py

```diff
@@ -260,14 +260,15 @@ def should_edit(attribute):
 
     if should_edit("name"):
         name = request.POST['name']
-        form.name[lang] = name
+        form.name[lang] = html_name(name)
```

Comment:
Agree that using `html_name` isn't the right thing here. I think you just want `html.strip_tags`